### ISSUE TITLE
Electron 9.00 - checkbox not using accent color on Mac

### ISSUE THREAD

**Issue by dgmid** [electron/electron#23768]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 9.0.0
* **Operating System:**
  * macOS 10.15.4
* **Last Known Working Electron version:**
  * 8.3.0

### Expected Behavior
checkboxes should use the accent color that is set in System Preferences > General

### Actual Behavior
checkboxes remain blue

### To Reproduce

using electron fiddle:
https://gist.github.com/e27800f409182981352b68a90690c4a6

### Screenshots

<img width="759" alt="checkbox-accent-color" src="https://user-images.githubusercontent.com/1267580/82912561-8a5fb080-9f6d-11ea-9612-26bedf702a3c.png">

**Comment by codebytere** [comment]
@dgmid looks like this was introduced by an upstream change - it's being tracked [here](https://bugs.chromium.org/p/chromium/issues/detail?id=1085380&q=input%20checkbox%20color&can=2). I'll keep this open for now but it's likely we'll get this fix for free when Chromium fixes it.

**Comment by jlongster** [comment]
I see this too, particularly the focus ring described here: https://github.com/electron/electron/issues/23853

I don't understand because it works fine in Chrome 87:

<img width="131" alt="Screen Shot 2020-12-16 at 11 24 44 AM" src="https://user-images.githubusercontent.com/17031/102376158-5394ca80-3f91-11eb-9764-765c7a47900f.png">

If I change the accent color in macOS system settings, it properly updates to use the right one. Based on the button design, you can see that this is using the updated form styling in Chrome.

This doesn't look like it's a bug in Chrome, while it still happens in Electron v11?

**Comment by erickzhao** [comment]
It seems like a fix for the upstream issue was landed in https://chromium.googlesource.com/chromium/src/+/a1def49a34fefb3865c638ae68527518aa2dadc0

OmahaProxy tells me that it first landed in version `92.0.4479.0`,

I don't see it working up to the Electron 15 nightlies (`93.0.4566.0`), though. 🤔

**Comment by codebytere** [comment]
This is a runtime enabled feature behind a flag - you can now make it work by instantiating your BrowserWindow like:

```
new BrowserWindow({
      vibrancy: 'sidebar',
      x: 200,
      y: 200,
      webPreferences: {
        enableBlinkFeatures: 'CSSAccentColor'
      }
})
```
